quickwit-cli/src/source.rs

```diff
@@ -35,22 +35,20 @@ pub fn build_source_command<'a>() -> Command<'a> {
     Command::new("source")
         .about("Manages sources.")
         .subcommand(
-            Command::new("add")
-                .about("Adds a new source.")
+            Command::new("create")
+                .about("Adds a new source to an index.")
                 .args(&[
-                    arg!(--config <CONFIG> "Quickwit config file").env("QW_CONFIG"),
-                    arg!(--index <INDEX> "ID of the target index"),
-                    arg!(--params <PARAMS> "Parameters for the source formatted as a JSON object passed inline or via a file. Parameters are source-specific. Please, refer to the source's documentation for more details."),
-                    arg!(--source <SOURCE_ID> "ID of the source."),
-                    arg!(--type <TYPE> "Type of the source. Available types are: `file` and `kafka`."),
+                    arg!(--config <CONFIG> "Path to Quickwit config file").env("QW_CONFIG"),
+                    arg!(--index <INDEX_ID> "ID of the target index"),
```

Comment:
I thought you were heading for index_id integrated in the source file?

Reply:
As we may revisit how we manage objects in a near future (https://github.com/quickwit-oss/quickwit/issues/1024), I chose to leave the index out of the source config so I did not have to write some custom parsing logic.